Goobi/src/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -74,7 +74,7 @@
 
 	/**
 	 * The field exportDmsTask holds an optional task instance whose progress
-	 * will be updated and whom errors will be passed to to be visible in the
+	 * will be updated and whose errors will be passed to be visible in the
```

Comment:
Your change appears to me to reverse the meaning. The errors will be passed into the the `exportDmsTask` instance. If I read ‘whose errors’ it more likely seems that the errors come from the the `exportDmsTask` which is not the case of useage here.

Maybe I got the translation wrong, but the intended meaning of the description is: _Das Feld exportDmsTask hält eine optionale Task-Instanz, deren Fortschritt aktualisiert wird und **der** Fehler übergeben werden um in der Anzeige des Task Managers sichtbar zu sein, wenn eine verfügbar ist._